tests/cli/run/tests/run_tests_api_28.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from core.base_test.tns_run_test import TnsRunTest
+from core.enums.platform_type import Platform
+from core.settings import Settings
+from core.utils.file_utils import Folder
+from data.sync.hello_world_js import sync_hello_world_js, run_hello_world_js_ts
+from products.nativescript.tns import Tns
+from products.nativescript.tns_paths import TnsPaths
+
+Settings.Emulators.DEFAULT = Settings.Emulators.EMU_API_28
+
+
+class TnsRunJSTests(TnsRunTest):
+    app_name = Settings.AppName.DEFAULT
+    source_project_dir = TnsPaths.get_app_path(app_name)
+    target_project_dir = os.path.join(Settings.TEST_RUN_HOME, 'data', 'temp', app_name)
+
+    @classmethod
+    def setUpClass(cls):
+        TnsRunTest.setUpClass()
+
+        # Create app
+        Tns.create(app_name=cls.app_name, template='tns-template-hello-world@6.0.2')
```

Comment:
Prefer `tns-template-hello-world@6.0`, this will capture 6.0.3 if we do hot fix release one day.

Reply:
agree